lib/modules/manager/pep723/schema.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod';
+import { Toml } from '../../../util/schema-utils';
+import { depTypes, pep508ToPackageDependency } from '../pep621/utils';
+
+const Pep723Dep = z
+  .string()
+  .transform((dep) => pep508ToPackageDependency(depTypes.dependencies, dep));
```

Comment:
`depType` will be `project.dependencies`, like default dependencies for PEP 621. Should we use a specific one for this manager?

Reply:
no, I think it's ok as is